### PR TITLE
chore: fix macos testing

### PR DESCRIPTION
**Pull request by eablack**
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001qz8gcYAA/view

node14 on macos-latest is broken with arm architecture. this fixes the problem by specifying x64.

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 16.19.0
```
Comment (sbosio) [inline .tool-versions:1]: Shouldn't we use 16.20.2 that is the last version that has all security issues patched?

File: package.json [hunk 2]
```diff
@@ -53,6 +53,17 @@
...
+    "require": [
+      "test/helpers/init.js",
+      "ts-node/register",
+      "source-map-support/register"
+    ],
+    "watch-extensions": "ts",
+    "recursive": true,
+    "reporter": "spec",
+    "timeout": 360000
+  },
```
Comment (sbosio) [inline package.json:66]: When adding the Mocha config options to `package.json`, shouldn't we remove `test/mocha.opts`? Or is it still required for Node.js 14?

**Review by sbosio — APPROVED** [review]
LGTM, but left a couple questions for you to review.

File: .tool-versions (hunk 1 above)
Comment (eablack) [reply .tool-versions]: nice, yeah, i just copied what we had in the main cli, but you're right. we should probably update the main cli as well.

File: package.json (hunk 2 above)
Comment (eablack) [reply package.json:66]: nope, just forgot to delete, thanks!

**Pull request merged**